nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py

```diff
@@ -187,6 +202,8 @@ def get_ipa_punctuation_list(locale):
     elif locale == "es-ES":
         # ref: https://en.wikipedia.org/wiki/Spanish_orthography#Punctuation
         punct_set.update(['¿', '¡'])
+    elif locale == "fr-FR":
+        punct_set.update(['–', '“', '”', '…', '̀', '́', '̂', '̈', '̧'])
```

Comment:
nit: do you have any reference on the punctuation set, like wikipedia or somewhere else?

Reply:
i extracted them from a large French TTS dataset we have, so no external reference for them!

Reply:
gotcha! thanks.